engines/lure/surface.cpp

```diff
@@ -467,13 +472,31 @@ Surface *Surface::newDialog(uint16 width, uint8 numLines, const char **lines, bo
 
 	Surface *s = new Surface(width, size.y);
 	s->createDialog();
+	#ifdef USE_TTS	
+	char *text = (char*) malloc(1024*sizeof(char));
+	memset(text, 0, 1024);
```

Comment:
Instead of using a char * here, you could use Common::String. The string manipulation code later on will be much easier and straightforward, and you won't need to have a hard limit of characters

Reply:
Done!